internal/recommendations/client.go

```diff
@@ -0,0 +1,176 @@
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen --config client.cfg.yml recommendations.v3.yml
```

Comment:
File internal/recommendations/recommendations.v3.yml looks empty in this version of the patch.